### DISCUSSION TITLE
Share public folder(s)

### DISCUSSION THREAD

**Discussion by pepew-le-boss** [vercel/turborepo#6200] [Help]
I have a turborepo and in the `apps` folder I have 2 **NextJS** apps. For each of these apps i have a `public` folder that have almost the same files.

I would like to know if there is a way to share these `public` folders into a single one ?
And if yes, how can it be achieved ?

**Comment by anthonyshew** [comment]
Going to close this as a duplicate of https://github.com/vercel/turbo/discussions/5046
